src/CommunicationLayer/CommunicationContainer.h

```diff
@@ -22,7 +20,7 @@ class CommunicationContainer
     I_JsonReceiver& jsonReceiver();
     ConnectionController& connectionController();
     InternetConnectionService& internetConnectionService();
-    CommDeviceManager& commDeviceManager();
+    InternetCommDevice& commDeviceManager();
```

Comment:
The variable name should be called `internetCommDevice_`, since it's of that type.